datadog/data_source_datadog_cloud_workload_security_agent_rules.go

```diff
@@ -12,7 +12,7 @@ import (
 
 func dataSourceDatadogCloudWorkloadSecurityAgentRules() *schema.Resource {
 	return &schema.Resource{
-		Description: "Use this data source to retrieve information about existing Cloud Workload Security Agent Rules for use in other resources.",
+		Description: "Use this data source to retrieve information about existing Cloud Workload Security Agent Rules for use in other resources (this data source is going to be deprecated soon).",
```

Comment:
Should we just call this "deprecated" rather than "going to be deprecated soon"?  I think that's more clear.